CHANGELOG.next.md

```diff
@@ -16,7 +16,7 @@ Thanks, you're awesome :-) -->
 
 #### Added
 
-* Added `search.*` fields #729
```

Comment:
Please add back this line

Reply:
so for future clarification - when I update from source, I should **just** add a line to CHANGELOG.next.md?

Reply:
Yes, just add your line and don't touch the rest :-)

Reply:
Looks the conflicts with the `CHANGELOG.next` are still outstanding.